android/build.gradle

```diff
@@ -35,11 +27,8 @@ buildscript {
     }
     dependencies {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${project.ext.kotlinPluginVersion}"
-        classpath "com.android.tools.build:gradle:${project.ext.gradlePluginVersion}"
+        classpath("com.android.tools.build:gradle")
```

Comment:
Why are we skipping the version?

Reply:
we no longer need to specify version, ref : https://github.com/facebook/react-native/blob/0.72-stable/packages/react-native/template/android/build.gradle#L18